src/Controls/tests/DeviceTests/Elements/Label/LabelTests.iOS.cs

```diff
@@ -18,5 +20,117 @@ public partial class LabelTests
 
 		int GetPlatformMaxLines(LabelHandler labelHandler) =>
  			(int)GetPlatformLabel(labelHandler).Lines;
+
+		double GetPlatformCharacterSpacing(LabelHandler labelHandler)
+		{
+			var attributedText = GetPlatformLabel(labelHandler).AttributedText;
+			var characterSpacing = attributedText.GetCharacterSpacing();
+			return characterSpacing;
+		}
+
+		double GetPlatformLineHeight(LabelHandler labelHandler)
+		{
+			var attributedText = GetPlatformLabel(labelHandler).AttributedText;
+			var lineHeight = attributedText.GetLineHeight();
+			return lineHeight;
+		}
+
+		TextDecorations GetPlatformTextDecorations(LabelHandler labelHandler)
+		{
+			var attributedText = GetPlatformLabel(labelHandler).AttributedText;
+			var textDecorations = attributedText.GetTextDecorations();
+			return textDecorations;
+		}
+
+		[Fact(DisplayName = "Using CharacterSpacing with LineHeight and TextDecorations works Correctly")]
+		public async Task CharacterSpacingWithLineHeightWithTextDecorationsWorksCorrectly()
```

Comment:
This test should probably be rewritten as a theory where you pass the expectedCharacterSpacing, expectedLineHeight, and expectedTextDecorations.

Reply:
Hi, @jknaudt21 
Thank you for reviewing the test code.
Changed the test code to use MemberData. Changed the test data to be given by the GetCharacterSpacingWithLineHeightWithTextDecorationsWorksTestData method.